pandas/core/internals/managers.py

```diff
@@ -1738,7 +1745,7 @@ def form_blocks(arrays, names, axes):
         block_type = get_block_type(v)
         items_dict[block_type.__name__].append((i, k, v))
 
-    blocks = []
+    blocks: List[Block] = []
```

Comment:
Just as FYI but newer versions of mypy technically have better inferencing capabilities for empty sequence assignment like this. @simonjayhawkins

Reply:
id be happy to remove this; is that the suggested course of action?